src/views/ChooseBond/BondRow.jsx

```diff
@@ -109,7 +109,7 @@ export function BondTableData({ bond }) {
       <TableCell align="center">
         <Typography>
           <>
-            <span class="currency-icon">{priceUnits(bond)}</span>{" "}
+            <span className="currency-icon">{priceUnits(bond)}</span>{" "}
```

Comment:
line 82 above! `state.bonding[bond].loading no longer exists`  so its always going to think its loading.

Reply:
Would using `state.bonding.status` be okay? it lumps _all_ bonds into a loading state.